rewrite-groovy/src/test/java/org/openrewrite/groovy/tree/MethodDeclarationTest.java

```diff
@@ -71,7 +71,7 @@ void methodDeclarationWithModifiers() {
         rewriteRun(
           groovy(
             """
-              def accept(final def Map m) {
+              public final accept(final def Map m) {
```

Comment:
Not sure if you actually intended to modify a test instead of adding a new one.

Reply:
Yep! But now I think about it again, let's create a new test instead of modifying this one. That way it's more explicit we can have modifiers on both the return type and the arguments.